drivers/hmis_external_apis/app/views/layouts/hmis_external_apis/external_forms.haml

```diff
@@ -70,6 +72,9 @@
       fieldset > .form-group:has(> .form-check) {
           margin-bottom: 0px;
       }
+      .leaflet-container {
+        height: 500px;
```

Comment:
Looking on mobile using dev tools, I would  suggest smaller (300px) - but we can see how it looks on a real device